wx_test_output.hpp

```diff
@@ -56,11 +56,6 @@ class output_file_existence_checker
         return fs::exists(path_);
         }
 
-    std::string const& path() const
```

Comment:
@thesiv Do you remember why did this commit have to be done?

It looks to me that just changing the return type here to `std::string` (i.e. returning by value) is enough to make the code compile and work and it seems preferable as it's a more minimal change.